settings.gradle.kts

```diff
@@ -0,0 +1,23 @@
+pluginManagement {
+    repositories {
+        maven("https://gradle.pkg.st")
+    }
+}
+
+plugins {
+    id("build.less") version "1.0.0-rc2"
+    id("org.gradle.toolchains.foojay-resolver-convention") version "0.7.0"
+}
```

Comment:
Here, we are installing some settings-time plugins.

- The [Buildless plugin for Gradle](https://github.com/buildless/plugin-gradle) offers optimized build caching via the [Buildless Cloud](https://less.build/resources/network), and via the [Buildless Agent](https://docs.less.build/docs/agent). If there is no Buildless API key or running agent, the plugin is inert, so installing it this way in CI won't break anything (that's intentional, to allow for smooth setup).

- The [Foojay Resolver plugin](https://github.com/gradle/foojay-toolchains) resolves JVM toolchains automatically, and can download JDKs and use them for multi-JDK testing. It's a helpful plugin for specifying a specific operating or tooling JVM without burden on the user to install / setup that version.